File: AkkaActors/Matches/Match.cs
namespace AkkaActors.Matches
{
    public class Match
    {
        public int Id { get; set; }
        public string Team1 { get; set; }
        public string Team2 { get; set; }
        public int Team1Score { get; set; }
        public int Team2Score { get; set; }
        public bool Started { get; set; }
        public bool Ended { get; set; }
    }
}

File: AkkaActors/ClientApp/src/app/models/match.ts
export interface Match {
  id: number;
  team1: string;
  team1Score: number;
  team2: string;
  team2Score: number;
  started: boolean;
  ended: boolean;
}

export interface MatchResponse {
  match: Match;
}

export interface MatchesResponse {
  matches: Match[];
}
